app/manage-projects/[projectSlug]/layout.tsx

```diff
@@ -86,6 +88,21 @@ function Safe({ children, projectSlug }: PropsWithChildren<{ projectSlug: string
     hasAlreadyClosedAlert.current = true;
   }
 
+  function renderUngrantButton() {
```

Comment:
Why did you put this in a function ? he can take place inside the renderActions callback

Reply:
Because I need to use it inside the financial detail sidepanel